bundles/org.openhab.binding.pihole/README.md

```diff
@@ -0,0 +1,169 @@
+Number clients_ever_seen "Clients Ever Seen" { channel="pihole:server:a4a077edb8:clients_ever_seen" }
+Number unique_clients "Unique Clients" { channel="pihole:server:a4a077edb8:unique_clients" }
+Number dns_queries_all_types "DNS Queries (All Types)" { channel="pihole:server:a4a077edb8:dns_queries_all_types" }
+Number reply_UNKNOWN "Reply UNKNOWN" { channel="pihole:server:a4a077edb8:reply_UNKNOWN" }
+Number reply_NODATA "Reply NODATA" { channel="pihole:server:a4a077edb8:reply_NODATA" }
+Number reply_NXDOMAIN "Reply NXDOMAIN" { channel="pihole:server:a4a077edb8:reply_NXDOMAIN" }
+Number reply_CNAME "Reply CNAME" { channel="pihole:server:a4a077edb8:reply_CNAME" }
+Number reply_IP "Reply IP" { channel="pihole:server:a4a077edb8:reply_IP" }
+Number reply_DOMAIN "Reply DOMAIN" { channel="pihole:server:a4a077edb8:reply_DOMAIN" }
+Number reply_RRNAME "Reply RRNAME" { channel="pihole:server:a4a077edb8:reply_RRNAME" }
+Number reply_SERVFAIL "Reply SERVFAIL" { channel="pihole:server:a4a077edb8:reply_SERVFAIL" }
+Number reply_REFUSED "Reply REFUSED" { channel="pihole:server:a4a077edb8:reply_REFUSED" }
+Number reply_NOTIMP "Reply NOTIMP" { channel="pihole:server:a4a077edb8:reply_NOTIMP" }
+Number reply_OTHER "Reply OTHER" { channel="pihole:server:a4a077edb8:reply_OTHER" }
+Number reply_DNSSEC "Reply DNSSEC" { channel="pihole:server:a4a077edb8:reply_DNSSEC" }
+Number reply_NONE "Reply NONE" { channel="pihole:server:a4a077edb8:reply_NONE" }
+Number reply_BLOB "Reply BLOB" { channel="pihole:server:a4a077edb8:reply_BLOB" }
+Number dns_queries_all_replies "DNS Queries (All Replies)" { channel="pihole:server:a4a077edb8:dns_queries_all_replies" }
+Number privacy_level "Privacy Level" { channel="pihole:server:a4a077edb8:privacy_level" }
+Switch enabled "Status" { channel="pihole:server:a4a077edb8:enabled" }
+String disable_enable "Disable Blocking" { channel="pihole:server:a4a077edb8:disable-enable" }
+```
+
+### Actions
+
+Pi-hole binding provides actions to use in rules:
+
+```java
+import java.util.concurrent.TimeUnit
+        
+rule "test"
+when
+    /* when */
+then
+	val actions = getActions("pihole", "pihole:server:as8af03m38")
+	if (actions !== null) {
+            // disable blocking for 5 * 60 seconds (5 minutes)
+            actions.disableBlocking(5 * 60)
+    
+            // disable blocking for 5 minutes
+            actions.disableBlocking(5, TimeUnit.MINUTES)
+    
+            // disable blocking for infinity
+            actions.disableBlocking(0)
+            actions.disableBlocking()
+    
+            // enable blocking
+            actions.enableBlocking()
+	} 
+end
+```
+
+## Support ❤️
+
+If you want to support the author of this binding, buy him a coffee:
+
+[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/S6S8UBWWY) <a href="https://buycoffee.to/magx2" target="_blank"><img src="https://buycoffee.to/btn/buycoffeeto-btn-primary.svg" alt="Postaw mi kawę na buycoffee.to" width="150"></a>
```

Comment:
See support comment @ salus binding. Sorry, but still no guidance here. For speed better to remove for now.

Reply:
Please remove for this to get merged.